packages/react/src/ReactElement.js

```diff
@@ -95,20 +95,20 @@ function defineRefPropWarningGetter(props, displayName) {
  * if something is a React Element.
  *
  * @param {*} type
+ * @param {*} props
  * @param {*} key
  * @param {string|object} ref
+ * @param {*} owner
  * @param {*} self A *temporary* helper to detect places where `this` is
  * different from the `owner` when React.createElement is called, so that we
  * can warn. We want to get rid of owner and replace string `ref`s with arrow
  * functions, and as long as `this` and owner are the same, there will be no
  * change in behavior.
  * @param {*} source An annotation object (added by a transpiler or otherwise)
  * indicating filename, line number, and/or other information.
- * @param {*} owner
- * @param {*} props
  * @internal
  */
-const ReactElement = function(type, key, ref, self, source, owner, props) {
+const ReactElement = function(type, props, key, ref, owner, self, source) {
```

Comment:
Ok, so I downloaded the artifacts. Somehow, this is not getting inlined anymore. That's why the file increases.

I'm not sure if it's called in more places and maybe it stops trying to inline it before it's able DCE the other paths.

It could also be the argument order. You could try restoring the argument order. It might have an effect on the compiler since the property reads of mutable values are not pure operations.

Reply:
Hey looks like reordering fixes it! Wild. Now there is only a size regression on WWW prod which is to be expected since the feature is turned on there.